src/runtime/virtcontainers/utils/utils_linux.go

```diff
@@ -100,11 +100,12 @@ const (
 	procDeviceIndex = iota
 	procPathIndex
 	procTypeIndex
+	procOptionsIndex
 )
 
-// GetDevicePathAndFsType gets the device for the mount point and the file system type
+// GetMountInfo gets the device for the mount point and the file system type
 // of the mount.
-func GetDevicePathAndFsType(mountPoint string) (devicePath, fsType string, err error) {
+func GetMountInfo(mountPoint string) (devicePath, fsType string, options []string, err error) {
```

Comment:
TBH, I'm not sure what's idiomatic here, but it seems this function is harder to maintain and use since it has _so_ many return values. Should we look at a struct? What do others think?